File: working/temas/31.2.1.rb
class Frigo
    attr_reader :temperatura

    def initialize(color, temperatura)
        @color = color
        @temperatura = temperatura
    end

    def temperatura=( nueva_temperatura )
        @temperatura = nueva_temperatura
    end
end


f1 = Frigo.new("rojo", 7)
f1.temperatura = 17
puts f1.temperatura

File: working/temas/32-visibilidad-atributos/31.2.1.py
class Frigo:
    def __init__(self,color,temperatura):
        self.__color = color
        self.__temperatura = temperatura

    @property
    def temperatura(self):
        return self.__temperatura

    def set_temperatura(self,nueva_temperatura):
        self.__temperatura = nueva_temperatura


f1 = Frigo("rojo", 7)
f1.set_temperatura(17)
print(f1.temperatura)